service-base/src/main/java/org/eclipse/hono/service/auth/device/ExecutionContextAuthHandler.java

```diff
@@ -61,25 +68,28 @@ public final AuthProvider getAuthProvider() {
     @Override
     public Future<DeviceUser> authenticateDevice(final T context) {
 
+        Objects.requireNonNull(context);
+
         final Promise<DeviceUser> result = Promise.promise();
         parseCredentials(context)
-                .compose(authInfo -> {
```

Comment:
It is hard for me to spot any change here. To me the code formatting looks off to what we had in the past. If we want to change the formatting, then I think we should agree on that, and make this a dedicated changed. Rather than making this as part of an actual PR. Or at least confine it to a dedicated commit.

Reply:
That is true, our current code formatting rules seem to indeed enforce the original indentation.
I will revert the change. In general, do you think it is a good idea to automatically format the code according to our formatter rules on each save in the IDE?